AzureMonitorAgent/agent.py

```diff
@@ -450,7 +450,11 @@ def uninstall():
         log_and_exit("Uninstall", UnsupportedOperatingSystem, "The OS has neither rpm nor dpkg" )
     hutil_log_info('Running command "{0}"'.format(AMAUninstallCommand))
 
-    remove_localsyslog_configs()
+    remove_localsyslog_configs()    
+
+    # remove the logrotate config
+    if os.path.exists('/etc/logrotate.d/azuremonitoragentextension'):      
```

Comment:
We should be using a constant value for this path/filename.

Reply:
added